FCWTAPI/CWTFrequencies.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FCWTNET
+{
+    public class CWTFrequencies
+    {
+        public double[]? WaveletCenterFrequencies { get; private set; }
+        public int Length { get; private set; }
+        
+        public CWTFrequencies(double[] centerFrequencies)
+        {
+            WaveletCenterFrequencies = centerFrequencies;
```

Comment:
Is this centerFrequencies the parameter that I would enter my frequency array in?